### DISCUSSION TITLE
NovelWriter review in PCLinuxOS Magazine

### DISCUSSION THREAD

**Discussion by sbstn87** [vkbo/novelWriter#1459] [General]
https://pclosmag.com/html/Issues/202306/page03.html

"Overall this is an excellent program for (aspiring) authors. The program appears instantly on your screen and is very well behaved. It has good configuration options, especially when related to the project at hand. It has a tendency to grow on you, and once you're familiar with it, you can be very productive with it. All that being said, its user interface can seem clunky at places."

**Comment by johnblommers** [comment]
> "...its user interface can seem clunky at places."

Not exactly a helpful comment; vague & non-specific. Good grief, this comment can apply to just about any application or GUI. MS Word has been roundly criticized since forever, especially its ribbon design. Vim versus Emacs anybody?

**Reply by vkbo** [reply]
I'm guessing they mean it has an old fashioned look perhaps? Which is true insofar as that is the standard look of Qt Widgets. I'm designing new ones or changing the default look in a lot of places.

It can also be referring to some features that are still in their original state. Version 1.0 of novelWriter was feature complete, but lacked polish because I first focused on adding all the necessary functionality. I'm working my way from component to component, but haven't gotten to everything yet. Next up is the Build Tool, and I'm finishing up the last bits now. But it's taken me half a year.
